### PR TITLE
Fix/notifications

### PR DESCRIPTION
**Pull request by melisabok**
Send `parentGroup` parameter when sending the reminders to the PCs can see all theses messages in the messages UI.

### CODE REVIEW

File: openreview/conference/templates/reviewerWebfield.js [hunk 1]
```diff
@@ -378,21 +378,3 @@ $('#group-container').on('click', 'a.note-contents-toggle', function(e) {
   var updated = $(this).text() === hiddenText ? visibleText : hiddenText;
   $(this).text(updated);
 });
-
-$('#group-container').on('click', 'a.send-reminder-link', function(e) {
```
Comment (zbialecki) [inline openreview/conference/templates/reviewerWebfield.js:382]: Why was this handler removed? Is it no longer being used?

**Review by zbialecki — COMMENTED** [review]
Looks good, just one question

File: openreview/conference/templates/reviewerWebfield.js (hunk 1 above)
Comment (melisabok) [reply openreview/conference/templates/reviewerWebfield.js:382]: do we send reminders from the reviewers console?

I think this was a copy/paste from the AC console, correct me if I'm wrong.

Comment (zbialecki) [reply openreview/conference/templates/reviewerWebfield.js:382]: I took at look and I think you are right.

**Review by zbialecki — APPROVED** [review]

**Review by celestemartinez — APPROVED** [review]

**Pull request merged**